rllib/policy/tf_policy.py

```diff
@@ -235,13 +230,22 @@ def __init__(
         self._update_ops = update_ops
         self._apply_op = None
         self._stats_fetches = {}
-        self._timestep = (
-            timestep
-            if timestep is not None
-            else tf1.placeholder_with_default(
-                tf.zeros((), dtype=tf.int64), (), name="timestep"
-            )
-        )
+
+        if timestep is None:
+            if hasattr(self, "timestep"):
+                timestep = self.timestep
+            else:
+                timestep = tf1.placeholder_with_default(
+                    tf.zeros((), dtype=tf.int64), (), name="timestep"
+                )
+        self._timestep = timestep
```

Comment:
assigning self.timestep to self._timestep?
can you add a comment about what you are doing here?

Reply:
Thanks, changed!